### ISSUE TITLE
Unity-Prod Shared Services common auth: Cognito Groups, Users (CS)

### ISSUE THREAD

**Issue by galenatjpl** [unity-sds/unity-cs#313]
Ensure that the shared services common Cognito users and groups exist and are configured in the `Unity-Prod` account.  The set of users should be those that are necessary to access SBG shared services and SBG venue services. 

Each user gets a Unity-SBG-Dev-ManagementConsole-ReadOnly role
One or more users get the Unity-SBG-Dev-ManagementConsole-Admin role

mod_auth_openidc adds request headers that represent the Cognito groups (one CSV header).

Acceptance Criteria:
- [ ] Inspect network traffic on MC, to make sure headers are passed in (on the SBG Dev MC instance)

**Comment by galenatjpl** [comment]
@ramesh-maddegoda please ensure each user in `Unity-Prod` gets a Unity-SBG-Dev-ManagementConsole-ReadOnly group.  And one or more special users (to be designated by @mike-gangl) get the Unity-SBG-Dev-ManagementConsole-Admin group.

**Comment by galenatjpl** [comment]
Blocked on getting the latest stable MC deployed to unity-venue-ops.  After that, @ramesh-maddegoda can verify the headers get passed.

**Comment by galenatjpl** [comment]
closing, since this will be covered in another ticket.
